AppSigner/MainView.swift

```diff
@@ -869,7 +869,12 @@ class MainView: NSView, URLSessionDataDelegate, URLSessionDelegate, URLSessionDo
                         if shouldSkipGetTaskAllow {
                             profile.removeGetTaskAllow()
                         }
-                        let isWildcard = profile.appID == "*" // TODO: support com.example.* wildcard
+                        var isWildcard = profile.appID == "*"
+                        if !isWildcard {
+                            if profile.appID.reversed().starts(with: "*.") { // support com.example.* wildcard
+                                isWildcard = true
+                            }
+                        }
```

Comment:
there's an easier way:

```suggestion
                        let isWildcard = profile.appID == "*" || profile.appID.suffix(2) == ".*"
```